src/StringToVectorConverter.hpp

```diff
@@ -1,35 +1,36 @@
-#ifndef STRINGTOVECTORCONVERTER_H
-#define STRINGTOVECTORCONVERTER_H
+#pragma once
 
 #include <string>
 #include <vector>
 #include <algorithm>
 
+int lambda(char c){
```

Comment:
By named lambda I meant a true lambda, not function :)

auto lambda = [vec](char c) { ... };
And it should stay in below function, just before the transform algorithm.